src/mono/wasm/Makefile

```diff
@@ -60,6 +60,7 @@ MONO_LIBS = \
 	$(MONO_BIN_DIR)/libmono-profiler-aot.a \
 	${NATIVE_BIN_DIR}/libSystem.Native.a \
 	${NATIVE_BIN_DIR}/libSystem.IO.Compression.Native.a \
+	${NATIVE_BIN_DIR}/libLdapProxy.Native.a \
```

Comment:
Same here - all changes under `src/mono` should be reverted. Ldap support should not be linked into the runtime in any way.